docs/assistant/cloud/libraries/adding-assets/assistant-cloud.md

```diff
@@ -0,0 +1,20 @@
+---
+id: assistant-cloud
+title: Adding Assets via the Assistant Pro Cloud
+sidebar_label: Assistant Pro Cloud
+---
+
+### How to ​​Create a New Library From Assistant Pro Cloud
```

Comment:
Title is "Adding Assets" but first procedure is how to create a new library--at least needs a conceptual intro about where you add assets.